File: project/springboot/open-api/src/main/java/com/openapi/domain/vo/AgentVo.java
package com.openapi.domain.vo;

import lombok.Data;
import org.jetbrains.annotations.Nullable;

/**
 * @author 13225
 * @date 2025/9/29 13:46
 */
@Data
public class AgentVo {
    // agent name
    public String name;
    // agent description
    public String description;
    // agent avatarUrl
    @Nullable
    public String avatarUrl = null;
}

File: app/android/app/src/main/java/com/magicvector/domain/vo/agent/AgentVo.kt
package com.magicvector.domain.vo.agent

/**
 * Agent Item vo
 */
data class AgentVo(
    // 名称/描述一般不修改 → 不可变val
    val name: String,
    val description: String,
    // 头像URL可能动态修改 → 可变var + 可空
    var avatarUrl: String? = null
)